src/main/scala/com/atomist/tree/treeNode.scala

```diff
@@ -75,6 +76,20 @@ trait TreeNode extends Visitable {
     */
   def childrenNamed(key: String): Seq[TreeNode]
 
+  /**
+    * Is this tree node here to help other nodes
+    * hang together, or does it have significance
+    * to the user and the outside world?
+    */
+  def significance: Significance = TreeNode.Undeclared
+
+}
+
+object TreeNode {
```

Comment:
I love the concept, but don't like `Structural` and `Explicit` for several reasons. 

- They are not opposites
- `Structural` sounds important, whereas in fact it's unimportant (padding etc.)

I suggest we change to `Noise` and `Signal`.